### ISSUE TITLE
Editor feature: Undo Ctrl/Cmd + Z

### ISSUE THREAD

**Issue by iaomw** [BoomingTech/Pilot#83]
It's common to have Undo feature in any editor. When I was playing with the demo, this feature seems missing.

However, it's not easy to build a robust undo feature while the engine is still evolving. It's good to have before next millstone.

**Comment by Jenyziyan89** [comment]
Only fans adult 🍌🍌🍌http://lnkiy.in/Jenyziyan

**Comment by hyv1001** [comment]
Yes, the undo/redo mechanism is very necessary, however, it may take a long time to implement an elegant and robust solution.
This feature is in our roadmap, but the priority is not very high. 
Currently, we are focusing on course-related feature development.
